R/redist.R

```diff
@@ -707,389 +798,267 @@ redist.combine <- function(savename, nsims, nloop, nthin, nunits, temper = 0
 #' nsims = 10000)
 #' }
 #' @export
-redist.mcmc <- function(adjobj, popvec, nsims, ndists = NULL, initcds = NULL,
+redist.mcmc <- function(algorithm = 'mcmc', adjobj, popvec, nsims, ndists = NULL, initcds = NULL,

```

Comment:
Let's maybe move the algorithm argument after `adjobj`, `popvec`, and `nsims`. We should also do `algorithm = c("mcmc", "anneal")` and use a `match.arg` on it.

Reply:
Commit: "move algorithm arg and run match.arg"